Integrations/MISP_V2/MISP_V2.py

```diff
@@ -844,14 +886,13 @@ def search(post_to_warroom: bool = True) -> Tuple[dict, Any]:
         'category',
         'org',
         'tags',
-        'from',
-        'to',
+        'date_from',
```

Comment:
for the integration this is still `to` and `from`.
you need to handle that (one way is to switch the list to a mapping)